### PR TITLE
chore: upgrade snakeyaml version

### PR DESCRIPTION
**Pull request by mbasiuk-talend**
**What is the current behavior?** (You can also link to an open issue here)
Old version of Snakeyaml

**What is the new behavior?**
Latest 1.26 version

One more change to https://github.com/Talend/components/pull/1617 PR - updating 
services/components-api-service-rest/pom.xml.

Since in https://github.com/Talend/components/pull/1617 build https://ci-common.datapwn.com/job/components_pull-request_master/490/consoleFull 1.17 version can also be found.

**Please check if the PR fulfills these requirements**

- [x] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [x] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [x] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: services/components-api-service-rest/pom.xml [hunk 1]
```diff
@@ -32,6 +32,7 @@
         <talend_docker_registry>registry.datapwn.com</talend_docker_registry>
 
         <groovy.version>2.5.6</groovy.version> <!-- spring boot sucks with its bom -->
+        <snakeyaml.version>1.26</snakeyaml.version>
```
Comment (coheigea) [inline services/components-api-service-rest/pom.xml:35]: Looks like we also have to add this change to services/components-api-service-rest-all-components/pom.xml

**Review by coheigea — CHANGES_REQUESTED** [review]

File: services/components-api-service-rest/pom.xml (hunk 1 above)
Comment (mbasiuk-talend) [reply services/components-api-service-rest/pom.xml:35]: Agree. Please check the latest commit.

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/491/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/491/)

**Review by coheigea — APPROVED** [review]

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/494/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/494/)

**Pull request merged**